cf-templates/stack.yaml

```diff
@@ -1292,7 +1403,6 @@ Resources:
 ## Lambda Get PublicIP Information
   GetEC2PublicIP:
       Type: AWS::Lambda::Function
-      DependsOn: ReverseProxyLaunchConfig
```

Comment:
Why is this dependency being removed?

Reply:
It was not a correct dependency. Rather I have put the dependency on ReverseProxyASG.

Reply:
```
  Lambdatrigger:
     Type: 'Custom::GetEC2PublicIP'
     DependsOn: ReverseProxyASG
     Properties:
       ServiceToken: !GetAtt 'GetEC2PublicIP.Arn'
       GetPublicIP: !Ref GetEC2PublicIP
```